sidebarsAstro.js

```diff
@@ -92,7 +92,16 @@ module.exports = {
         "configure-worker-queues",
         "api-keys",
         "environment-variables",
-        "secrets-backend",
+        {
+          type: "category",
+          label: "Configure a secrets backend",
```

Comment:
```suggestion
          type: "category",
          label: "Configure a secrets backend",
          link: { type: 'doc', id: 'secrets-backend' },
```
Might want to check the behavior of this in preview, you may need to remove the overview doc from the items list.